qa/L0_optuna_search/test.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source ../common/util.sh
+source ../common/check_analyzer_results.sh
+create_logs_dir "L0_optuna_search"
+
+# Set test parameters
+MODEL_ANALYZER="`which model-analyzer`"
+REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
+MODEL_REPOSITORY=${MODEL_REPOSITORY:="/opt/triton-model-analyzer/examples/quick-start"}
+QA_MODELS="add_sub"
+MODEL_NAMES="$(echo $QA_MODELS | sed 's/ /,/g')"
+TRITON_LAUNCH_MODE=${TRITON_LAUNCH_MODE:="local"}
+CLIENT_PROTOCOL="grpc"
+PORTS=(`find_available_ports 3`)
+GPUS=(`get_all_gpus_uuids`)
+OUTPUT_MODEL_REPOSITORY=${OUTPUT_MODEL_REPOSITORY:=`get_output_directory`}
+CONFIG_FILE="config.yml"
+FILENAME_SERVER_ONLY="server-metrics.csv"
+FILENAME_INFERENCE_MODEL="model-metrics-inference.csv"
+FILENAME_GPU_MODEL="model-metrics-gpu.csv"
+
+rm -rf $OUTPUT_MODEL_REPOSITORY
+create_result_paths
+
+python3 test_config_generator.py --profile-models $MODEL_NAMES
+
+# Run the analyzer and check the results
+RET=0
+
+set +e
+
+MODEL_ANALYZER_ARGS="-m $MODEL_REPOSITORY -f $CONFIG_FILE"
```

Comment:
Would it be possible to make this a multi-line command where you're only setting MODEL_ANALYZER_ARGS once, or is this because Bash won't process them in a cleanly formatted multi-line statement (i.e. Bash needs this to be on one line)?